apps/web/src/app/[locale]/(dashboard)/[wsId]/calendar/client.tsx

```diff
@@ -27,11 +31,57 @@ export default function CalendarClientPage({
   const t = useTranslations('calendar');
   const locale = useLocale();
   const [isAddEventDialogOpen, setIsAddEventDialogOpen] = useState(false);
+  const [eventDetailsDialog, setEventDetailsDialog] = useState<{
+    isOpen: boolean;
+    event: WorkspaceScheduledEventWithAttendees | null;
+    isLoading?: boolean;
+  }>({
+    isOpen: false,
+    event: null,
+    isLoading: false,
+  });
+
+  const fetchEventDetails = async (
+    eventId: string,
+    scheduledEvent?: WorkspaceScheduledEventWithAttendees
+  ) => {
+    // If we already have the event data, use it directly
+    if (scheduledEvent) {
+      setEventDetailsDialog({
+        isOpen: true,
+        event: scheduledEvent,
+        isLoading: false,
+      });
+      return;
+    }
+
+    // Otherwise, fetch the event data
+    setEventDetailsDialog({ isOpen: true, event: null, isLoading: true });
+
+    try {
+      const response = await fetch(
+        `/api/v1/workspaces/${workspace.id}/scheduled-events/${eventId}`
+      );
+
+      if (!response.ok) {
+        throw new Error('Failed to fetch event details');
+      }
+
+      const event = await response.json();
+      setEventDetailsDialog({ isOpen: true, event, isLoading: false });
+    } catch (error) {
+      toast.error('Error fetching event details:');
+      console.error('Error fetching event details:', error);
+      setEventDetailsDialog({ isOpen: false, event: null, isLoading: false });
+      // Optionally show a toast error message here
+    }
+  };
 
   const extras =
     workspace.id === ROOT_WORKSPACE_ID ? (
       <div className="grid w-full items-center gap-2 md:flex md:w-auto">
         <AddEventButton onOpenDialog={() => setIsAddEventDialogOpen(true)} />
+        <CreateScheduledEventButton wsId={workspace.id} />
```

Comment:
Scheduled event and event is still the same thing right? Since an event can be created for a future use.